trade-finance-manager-ui/server/controllers/case/amendments/amendmentAnswers.controller.js

```diff
@@ -1,21 +1,22 @@
 const { format, fromUnixTime, getUnixTime } = require('date-fns');
-const { isTfmFacilityEndDateFeatureFlagEnabled } = require('@ukef/dtfs2-common');
 const { AMENDMENT_STATUS } = require('@ukef/dtfs2-common');
+const { HttpStatusCode } = require('axios');
 const api = require('../../../api');
 const { formattedNumber } = require('../../../helpers/number');
+const { isFacilityEndDateEnabledForTfmFacility } = require('../../helpers/isFacilityEndDateEnabledForTfmFacility');
 
 const getAmendmentAnswers = async (req, res) => {
   const { facilityId, amendmentId } = req.params;
   const { userToken } = req.session;
   const { data: amendment, status } = await api.getAmendmentById(facilityId, amendmentId, userToken);
+  const { dealId, requireUkefApproval, changeCoverEndDate, changeFacilityValue } = amendment;
+  const facility = await api.getFacility(facilityId, userToken);
```

Comment:
can an empty line be inserted here please

Reply:
Sure, have changed